### PR TITLE
Add missing unit test

### PR DESCRIPTION
**Pull request by sabahirfan**
### Change description ###
This PR adds missing unit test for JsonMapper class.

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java [hunk 1]
```diff
@@ -143,4 +150,38 @@ public void shouldThrowExceptionOnInvalidJsonWithTypeReference() {
...
+
+        Map<String, Object> data = new HashMap<>();
+        data.put("id", "1");
+        data.put("referenceNumber", "000MC001");
+        data.put("submitterId", "2");
+        data.put("submittedOn", timestamp);
+        data.put("externalId", uuid);
+        data.put("issuedOn", date);
+        data.put("responseDeadline", date.plusDays(14));
+        data.put("moreTimeRequested", "NO");
```
Comment (timja-kainos) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java]: YesNoOption.NO.name() ?

File: src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java [hunk 2]
```diff
@@ -143,4 +150,38 @@ public void shouldThrowExceptionOnInvalidJsonWithTypeReference() {
         processor.fromJson("{asads:", new TypeReference<List<Response>>() {
         });
     }
+
+    @Test
+    public void shouldConvertMapToObject() {
```
Comment (timja-kainos) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java]: seems to be converting to a ccdcase not an object here

Comment (sabahirfan) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java]: done

File: src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java (hunk 1 above)
Comment (sabahirfan) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java]: done

File: src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java [hunk 3]
```diff
@@ -143,4 +150,38 @@ public void shouldThrowExceptionOnInvalidJsonWithTypeReference() {
...
+            .referenceNumber("000MC001")
+            .submitterId("2")
+            .submittedOn(timestamp.format(ISO_DATE_TIME))
+            .externalId(uuid)
+            .issuedOn(date.format(ISO_DATE))
+            .responseDeadline(date.plusDays(14))
+            .moreTimeRequested(CCDYesNoOption.NO)
+            .build();
+
+        assertThat(ccdCase).isNotNull().isEqualTo(expected);
```
Comment (yarpo) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java]: do we need to check if `isNotNull`?

Comment (timja-kainos) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java]: seems overkill

Comment (sabahirfan) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/processors/JsonMapperTest.java]: Agree, it's not required

**Pull request merged**